doc/uftrace.md

```diff
@@ -356,7 +356,7 @@ record and `replay` internally. Thus, it describes most regular option in detail
 :   Hide small functions run less than the *TIME*
 
 \--task
-:   [info]: Print task relationship in a tree form instead of the tracing info.
+:   [info] Print task relationship in a tree form instead of the tracing info.
```

Comment:
I found that the same problem was there and the `[info]:` part was removed by 5c307433aa34d7788382e67423443565118cb0d6.

This is better to be removed.
```suggestion
:   Print task relationship in a tree form instead of the tracing info.
```